profile/README.md

```diff
@@ -0,0 +1,12 @@
+<div align="center">
+
+![Side Logo](./logo.svg)
+
+At Side, we’re **rebels with a cause**. <br>
```

Comment:
Like the copy - did you come up with that or see it somewhere?

Reply:
I copied it out of the header on our company about page 😁 

https://www.side.com/about/